src/NUnitFramework/nunitlite/OutputWriters/NUnit3XmlOutputWriter.cs

```diff
@@ -112,13 +112,12 @@ private TNode MakeTestRunElement(ITestResult result)
             testRun.AddAttribute("end-time", result.EndTime.ToString("u"));
             testRun.AddAttribute("duration", result.Duration.ToString("0.000000", NumberFormatInfo.InvariantInfo));
 
-            testRun.AddAttribute("total", (result.PassCount + result.FailCount + result.SkipCount + result.InconclusiveCount).ToString());
+            testRun.AddAttribute("total", result.TotalCount.ToString());
```

Comment:
Where's the Warning count?

Reply:
This is the old code. I changed it to result.totalcount

Reply:
Are you saying the green is not the latest code? It's what's in the PR.

Reply:
Green is the latest code only. I a have added warnings to total count at only single place, line no. 270 in this file-> https://github.com/nunit/nunit/blob/c39456ddc79c004fdcf8121e76d87d063a8c1e38/src/NUnitFramework/framework/Internal/Results/TestSuiteResult.cs

Reply:
I meant the warning element in the XML. Isn't that part of this?

Reply:
Ohh yeah, the warning attribute is missing here. I will add it and update you

Reply:
@CharliePoole - Added the warnings attribute - line number 120.